index.rst

```diff
@@ -131,8 +131,8 @@ special characters for wildcard matching:
 The backslash character (``\\``) can be used to escape a character so it is
 not interpreted as a special character.
 
-The maximum length of a section name is 4096 characters. All sections
-exceeding this limit are ignored.
+The minimum supported section name length is 1024 characters.
```

Comment:
To a new reader, this may sound like each key has to be at least 1024 chars long. I wonder if something like the following would be better 

```suggestion
Cores must accept section names with length up to 1024 characters.
```

Reply:
Tre tricky part here is that we have to write for two types of readers: editor config users and core/plugin devs. .)

Your suggestion hits the swet spots. I'd say. Thank you once more.